alerting/src/alert-worker.ts

```diff
@@ -40,21 +44,28 @@ async function processAlert(alert: Alert, alertRepository: Repository<Alert>) {
   url.searchParams.append('deactivate', 'true');
   url.searchParams.append('email', email);
 
+  const urlWithParams = formatUrl(prismUrl, {
+    hazardLayerIds: id,
+    date: maxDate.toISOString().slice(0, 10),
+  });
+
   if (alertMessage) {
     const emailMessage = `
         Your alert${alertName ? ` ${alertName}` : ''} has been triggered.
 
         Layer: ${serverLayerName}
         Date: ${maxDate}
 
-        Go to ${prismUrl} for more information.
+        Go to ${urlWithParams} for more information.
   
         Alert: ${alertMessage}`;
 
-    const emailHtml = `${emailMessage.replace(
-      /(\r\n|\r|\n)/g,
-      '<br>',
-    )} <br><br>To cancel this alert, click <a href='${url.href}'>here</a>.`;
+    const emailHtml = `${emailMessage
+      .replace(/(\r\n|\r|\n)/g, '<br>')
+      .replace(
+        urlWithParams,
+        `<a href="${urlWithParams}">${title}</a>`,
```

Comment:
Could we simply insert this in the string formatting above instead of doing a replacement?

Reply:
We can't insert in the string link with title different from URL